flask_project/campaign_manager/data_providers/osmcha_provider.py

```diff
@@ -0,0 +1,81 @@
+__author__ = 'Irwan Fathurrahman <irwan@kartoza.com>'
+__date__ = '29/05/17'
+
+from urllib import request
+from bs4 import BeautifulSoup
+from urllib.error import HTTPError, URLError
+from campaign_manager.data_providers._abstract_data_provider import (
+    AbstractDataProvider
+)
+
+
+class OsmchaProvider(AbstractDataProvider):
+    """Data from osmcha"""
+    osmcha_url = "https://osmcha.mapbox.com/?bbox=%(BBOX)s&page=%(PAGE)s"
```

Comment:
So this insight function only works for bbox area? not polygon area? 
Maybe we should add information about that in the view? Because what happen if a user create a polygon with a shape like triangle?